README.md

```diff
@@ -166,6 +166,40 @@ or:
 | `exact`     |          | `false`       | When `true`, will only match if `path` matches the URL exactly.                                                                                                                                                                                                                                             |
 | `strict`    |          | `false`       | When `true`, a `path` that has a trailing slash will only match a URL with a trailing slash.                                                                                                                                                                                                                |
 
+
+#### `link.js`
+
+An action used on anchor tags to navigate around the application. You can add an attribute `replace` to replace the current entry in the history stack instead of adding a new one.
+
+```html
+<a href="/" use:link>Home</a>
+<a href="/replace" use:link replace>Replace this URL</a>
+
+<script>
+  import link from 'svelte-routing/link';
+
+  export default {
+    actions: { link }
+  };
+</script>
+```
+
+#### `links.js`
+
+An action used on a root element to make all relative anchor elements navigate around the application. You can add an attribute `replace` on any anchor to replace the current entry in the history stack instead of adding a new one. You can add an attribute `noroute` for this action to skip over the anchor and allow it to use the native browser action.
+
+```html
+<!-- App.html -->
+<div use:links>
+  <MyComponent/>
+</div>
+
```

Comment:
Great stuff.
Should we show the import from `svelte-routing/links` here as well, like in the `link.js` example above?